README.md

```diff
@@ -14,6 +14,7 @@ largest and fastest growing open source group chat project.
 [![docs](https://readthedocs.org/projects/zulip/badge/?version=latest)](https://zulip.readthedocs.io/en/latest/)
 [![Zulip chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://chat.zulip.org)
 [![Twitter](https://img.shields.io/badge/twitter-@zulip-blue.svg?style=flat)](https://twitter.com/zulip)
+[![community](https://img.shields.io/badge/dynamic/json.svg?label=Community&colorB=%23FFA500&prefix=&suffix=online%20users&query=$.number_users&uri=https%3A%2F%2Fchat.zulip.org%2Fbadge%2Fusers)](https://chat.zulip.org/#)
```

Comment:
This should replace the "Zulip chat" badge above. Also, I think the wording can be
[Zulip chat | 4K users]
rather than Community.

Reply:
I would also simplify the URL as much as possible. So e.g. we don't need `&prefix=` if it's not being set. And does the uri have to escape the html?

Reply:
Oh, so instead of a dynamic button, it should be settled as a static one with 4K as a predefined number??
Doing this change from dynamic to static this will already happen, since the static url is as short as the other ones above the added button. That's the theory haha.
In case it's still a dynamic one, not quite sure of how to make the url more simplified...but I can look upon that :) .

Reply:
oh sorry, I do mean to keep it dynamic; 4K was just an example number.

Reply:
Oh, ok, I got it haha.
Make sense, I'll do that :)